e2e-tests/test/evm-apis.test.ts

```diff
@@ -130,3 +130,22 @@ describe("evm_revert", function () {
     expect(reverted).to.be.true;
   });
 });
+
+describe("evm_setBlockGasLimit", function () {
+  it("Should change block gas limit", async function () {
+    // Arrange
+    const wallet = new Wallet(RichAccounts[0].PrivateKey, provider);
+
+    // Act
+    const blockBefore = await provider.getBlock("latest");
+    const limitBefore = BigNumber.from(blockBefore.gasLimit).toString();
```

Comment:
lets check that `limitBefore` !=  "0x1000000000000"

Reply:
Isn't a stricter check preferable?